Sources/SnapshotTestCase.swift

```diff
@@ -92,8 +92,15 @@ open class SnapshotTestCase : XCTestCase {
     }
     
     private func recordSnapshot(of snapshotable: Snapshotable, options: Options, functionName: String, file: StaticString, line: UInt) throws {
-        try coordinator.recordSnapshot(of: snapshotable, options: options, functionName: functionName, line: line)
-        XCTFail("🔴 RECORD MODE: Reference image saved.", file: file, line: line)
+        let path = try coordinator.recordSnapshot(of: snapshotable, options: options, functionName: functionName, line: line)
+        XCTFail("🔴 RECORD MODE: Reference image saved to \(path.absoluteStringWithoutScheme)", file: file, line: line)
     }
 
 }
+
+fileprivate extension URL {
+    
+    var absoluteStringWithoutScheme: String {
+        return "/" + pathComponents.dropFirst().joined(separator: "/")
+    }
```

Comment:
I decided to put this here instead of in `SnapshotCoordinator` because I thought it's better to return the reference file path in it's URL form and let whoever cares format it. But that also means it's not tested. What do you think?